3party/succinct/bit_vector.hpp

```diff
@@ -41,7 +41,7 @@ namespace succinct {
         bit_vector_builder(uint64_t size = 0, bool init = 0)
```

Comment:
```suggestion
        bit_vector_builder(uint64_t size = 0, bool init = false)
```

Reply:
I've also renamed it to `initBit` because in my mind `bool init` means "whether to init this or not" instead of "what value to use to init"